iscript/src/iscript/macvpn.py

```diff
@@ -64,20 +65,123 @@ async def _sign_app(config, sign_config, app, entitlements_url, provisionprofile
     """
     # We mock the task for downloading entitlements and provisioning profiles
     entitlements_path = await download_entitlements_file(config, sign_config, {"payload": {"entitlements-url": entitlements_url}})
+
+    # TODO: Add provisionprofile_dir to scriptworker config
+    # https://github.com/mozilla-releng/scriptworker/blob/master/src/scriptworker/constants.py#L34
     provisioning_profile_path = None
-    if config.get("provisionprofile_dir"):
-        provisioning_profile_path = os.path.join(config["provisionprofile_dir"], provisionprofile_filename)
+
+    if provisionprofile_filename:
+        provisioning_profile_path = os.path.join("/builds/scriptworker/provisionprofiles", provisionprofile_filename)
```

Comment:
Why does this need to be hardcoded again?

Reply:
Ideally we'd add it to the [schema](https://github.com/mozilla-releng/scriptworker/blob/master/src/scriptworker/constants.py#L63) and modify script_config.yml with it. I'll make this a "default" instead